### PR TITLE
Feature/jupiter websocket

### PR DESCRIPTION
**Pull request by anggeloexe**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ services:
     ports:
       - "4000:4000"
     restart: always
-    command: npm run dev
+    command: pm2-runtime server.js -- --max-old-space-size=4096 NODE_ENV=production --trace-warnings
```
Comment (isaiastack) [inline docker-compose.yml]: I made some changes to devops flow. This change is not needed.

**Review by isaiastack — CHANGES_REQUESTED** [review]
@joramirezStackit please check the comment in the docker-compose.yml

File: docker-compose.yml (hunk 1 above)
Comment (anggeloexe) [reply docker-compose.yml]: sure, I'm going to revers this change

**Review by isaiastack — APPROVED** [review]
Looks good

**Pull request merged**